### PR TITLE
FDG-9491 Events udpate

### PR DESCRIPTION
**Pull request by chasls2**
https://federal-spending-transparency.atlassian.net/browse/FDG-9491

no change in test coverage

### CODE REVIEW

File: static/data/ga-event-tracking/spendingExplainerEventTracking.csv [hunk 1]
```diff
@@ -28,9 +28,9 @@
 27,Click on the link to the github repo,Fiscal Data - Explainers,Spending Citation Click,Spending,
 28,Click on the link to the Monthly Treasury Statement,Fiscal Data - Explainers,Spending Citation Click,Spending,
 29,Click on the link to the Interest Expense on Debt Outstanding,Fiscal Data - Explainers,Spending Citation Click,Spending,
-30,Click on the link to the Daily Treasury Statement,Fiscal Data - Explainers,Spending Citation Click,Spending,
+30,Click on the link to the Daily Treasury Statement,Fiscal Data - Explainers,Spending Citation Click,Monthly Treasury Statemen,
```
Comment (jdach) [inline static/data/ga-event-tracking/spendingExplainerEventTracking.csv:31]: Statement is missing a 't'

File: src/layouts/explainer/sections/federal-spending/how-much-does-the-govt-spend/how-much-does-the-govt-spend.jsx [hunk 2]
```diff
@@ -150,8 +150,11 @@ const HowMuchDoesTheGovtSpend = () => {
   const footer = (
     <div className={footerStyle}>
       Please note: Values displayed are outlays, which is money that is actually paid out by the government. Other sources, such as
-      <CustomLink url="https://www.usaspending.gov/"> USAspending</CustomLink>, may display spending as obligations, which is money that is promised
-      to be paid, but may not yet be delivered.
+      <CustomLink url="https://www.usaspending.gov/" eventNumber="34">
+        {' '}
```
Comment (jdach) [inline src/layouts/explainer/sections/federal-spending/how-much-does-the-govt-spend/how-much-does-the-govt-spend.jsx:154]: Space should be moved outside the link

File: static/data/ga-event-tracking/spendingExplainerEventTracking.csv [hunk 3]
```diff
@@ -11,9 +11,9 @@
 10,"Click on the ""Share this page:"" options",Fiscal Data - Explainers,Share Click,Spending - Share on <title of sharing option>,
 11,Click on the link to USAspending.gov,Fiscal Data - Explainers,Spending Citation Click,Spending,
 12,"Click on the button labeled ""Agency"" within the chart",Fiscal Data - Explainers,Chart Toggle,Spending - Spending Categories,
-13,"Click on the toggle to ""Dollars""",Fiscal Data - Explainers,Chart Toggle,Spending - Spending Categories,
-14,Click on the link to USAspending.gov Spending Explorer,Fiscal Data - Explainers,Spending Citation Click,Spending,
-15,Click on the link to Monthly Treasury Statement,Fiscal Data - Explainers,Chart Toggle,Spending - Spending Categories,
+13,"Click on the toggle to ""Dollars""",Fiscal Data - Explainers,Chart Toggle,Spending
```
Comment (jdach) [inline static/data/ga-event-tracking/spendingExplainerEventTracking.csv:14]: Trailing comma is missing

**Review by jdach — CHANGES_REQUESTED** [review]
National Debt Page
- The 'What is the National Debt?' Debt to the Penny link should have Debt to the Penny set as the eventLabel
Spending Explainer Page
- Agency Profile event should have 'Agency Profile' as the event label instead of 'Agency Profiles'
- MTS Link on the U.S. Government spending chart has "Spending - Spending Categories" as the eventLabel but it should be 'Monthly Treasury Statement (MTS)'

**Comment by nicweidner** [comment]
Updated Agency Profile! 
image

**Comment by nicweidner** [comment]
Updated MTS
image

**Review by jdach — APPROVED** [review]
Changes look good!

**Pull request merged**